hypothesis-python/tests/cover/test_lookup.py

```diff
@@ -112,7 +115,10 @@ def test_typing_Type_Union(ex):
 @pytest.mark.parametrize(
     "typ",
     [
-        collections.abc.ByteString,
+        pytest.param(
+            getattr(collections.abc, "ByteString", ...),
+            marks=pytest.mark.skipif(sys.version_info[:2] == (3, 14), reason="removed"),
```

Comment:
I see this is work in progress, but
```suggestion
            marks=pytest.mark.skipif(sys.version_info[:2] >= (3, 14), reason="removed"),
```
?

Reply:
Yeah, looks like I just changed the wrong comparison late at night 🤦‍♂️